tasks/main.yml

```diff
@@ -1,2 +1,19 @@
 ---
+- name: Create script from template
+  template:
+    src: efi-clone.sh.j2
+    dest: /usr/local/bin/efi-clone.sh
+    mode: 0755
 
+- name: Create Service from template
+  template:
+    src: efi-clone.service.j2
+    dest: /etc/systemd/system/efi-clone.service
+  notify: reload systemd
+
+
```

Comment:
Why two empty lines?

Reply:
That's just a formatting error. I've fixed it.